man/systemd-system.conf.xml

```diff
@@ -361,8 +361,10 @@
         <listitem><para>Configure the default value for the per-unit <varname>TasksMax=</varname> setting. See
         <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>
         for details. This setting applies to all unit types that support resource control settings, with the exception
-        of slice units. Defaults to 15%, which equals 4915 with the kernel's defaults on the host, but might be smaller
-        in OS containers.</para></listitem>
+        of slice units. Defaults to 15% of the kernel's option <varname>kernel.pid_max=</varname> or root cgroup <varname>"pids.max"</varname>.
+        Kernel has a default value for <varname>kernel.pid_max=</varname> and an algorithm of counting in case of more than 32 cores.
```

Comment:
`default value`? Maybe, `Kernel has a command line option <varname>kernel.pid_max=</varname>`. No? Though, I am not sure it is a kernel command line option or not...

Reply:
This is not a command-line option, just a kernel option controlled through `sysctl`.

Reply:
Do I need to provide here default value from kernel for `kernel.pid_max`?

Reply:
> This is not a command-line option, just a kernel option controlled through `sysctl`.

Oh, thanks. I did not know that. So, I think `kernel's option` -> `sysctl setting` is better in the previous line.

> Do I need to provide here default value from kernel for `kernel.pid_max`?

I think it is not necessary.

Reply:
I think yes, so I've changed that

Reply:
Sysctls can be also set with kernel command line options, for example `sysctl.kernel.pid_max=12345`.

Reply:
Yes, starting from 5.8 only.